### PR TITLE
#56: Add common project workflows

### LINKED ISSUE #56: Add common project workflows
## Background
Automation and checks of the project.

## Feature
Add the common workflows used in other similar projects.

### PR DESCRIPTION
**Pull request by benedeki**
* Added license_check.yml
* Added dependent_items.yml
* Renamed assign_project_to_issue.yml to assign_issue_to_project.yml
* Added codeowners

Closes #56

### CODE REVIEW

File: .github/workflows/license_check.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#
+
+name: License Check
+
+on:
+  push:
+    branches: [ main, develop, master ]
+  pull_request:
+    branches: [ master ]
+    types: [ assigned, opened, synchronize, reopened, labeled ]
```
Comment (miroslavpojer) [inline .github/workflows/license_check.yml:24]: This looks to much wide.

In Atum there is only:
`on:`
`  push:`
`    branches: [ master, develop ]`
`  pull_request:`
`    branches: [ master, develop ]`

**Review by miroslavpojer — COMMENTED** [review]
- [x] pulled
- [x] built
- [ ] tests run
- [ ] code reviewed

I am missing workflow for jacoco code measuring.

**Comment by miroslavpojer** [comment]
A command `sbt test` from root location does not run any test from project modules.
A command `sbt test` from **{root}/agent** location leads to 25 errors.
A command `sbt test` from **{root}/agent** location leads to 100 errors.

Did I miss some specific way how to run tests? In Read me files there is not useful information about this topic.

**Comment by miroslavpojer** [comment]
I am missing workflow for jacoco code measuring.

File: .github/workflows/license_check.yml (hunk 1 above)
Comment (benedeki) [reply .github/workflows/license_check.yml:24]: Good point, actually even `develop` is not present in the moment ;-)

**Comment by benedeki** [comment]
> A command `sbt test` from root location does not run any test from project modules. A command `sbt test` from **{root}/agent** location leads to 25 errors. A command `sbt test` from **{root}/agent** location leads to 100 errors.
> 
> Did I miss some specific way how to run tests? In Read me files there is not useful information about this topic.

> I am missing workflow for jacoco code measuring.

Those to come in subsequent PRs. Originally planned to be all in one, but as it has its own complication it's planned for #64

**Review by miroslavpojer — APPROVED** [review]

**Pull request merged**